components/upload/text/BottomRow.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Thank for organizing these components! Very Nitpicky, but I think It would be good however to organize these components within the upload directory so that we know in the future to navigate to it whenever we do need to change something easily!